src/apps/filters/h3.c

```diff
@@ -121,12 +156,19 @@ bool generalHelp(int argc, char *argv[]) {
         .helpText =
             "Convert degrees latitude/longitude coordinate to an H3 cell.",
     };
-    Arg *args[] = {&helpArg, &cellToLatLngArg, &latLngToCellArg};
+    Arg cellToBoundaryArg = {
+        .names = {"cellToBoundary"},
+        .helpText =
+            "Convert an H3 cell to an array of latitude/longitude coordinates "
+            "defining its boundary",
+    };
+    Arg *args[] = {&helpArg, &cellToLatLngArg, &latLngToCellArg,
+                   &cellToBoundaryArg};
     const char *helpText =
         "Please use one of the subcommands listed to perform an H3 "
         "calculation. Use h3 <SUBCOMMAND> --help for details on the usage of "
         "any subcommand.";
-    return parseArgs(argc, argv, 3, args, &helpArg, helpText);
+    return parseArgs(argc, argv, 4, args, &helpArg, helpText);
```

Comment:
4 is the number of arguments? Maybe add a named constant or comment here to remind us what this is.

Reply:
Yes, it's basically the length of the `args` array, and it would be nice to not need to provide it, but I'll at least make it a CONSTANT to be clearer, since I missed needing to change this at first.